src/EhsnPlugin/DataModel/ParsedEhsnLevelSurvey.cs

```diff
@@ -35,14 +35,14 @@ public class ParsedEhsnLevelSurvey
             LevelSummaryRows = new List<EHSNLevelNotesLevelChecksSummaryTableRow>(summaryTableRows);
         }
 
-        private const string PrimaryPrefix = "**";
+        private const string PrimaryPrefix = "*";
 
         public static string SanitizeBenchmarkName(string value)
         {
-            if (value == null || !value.StartsWith(PrimaryPrefix))
+            if (value == null || !value.Contains(PrimaryPrefix))
                 return value;
 
-            return value.Substring(PrimaryPrefix.Length).Trim();
+            return value.Replace("*", "");
```

Comment:
You probably want `.Replace("*", string.Empty).Trim()`, since I recall there are some EHSN files with trailing whitespace (humans are goofy like that).